README.md

```diff
@@ -1,5 +1,6 @@
 # Lavinia-client
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/64b563a2c5e349e2a13fcfe3c1bc1008)](https://www.codacy.com/gh/Project-Lavinia/Lavinia-client?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Project-Lavinia/Lavinia-client&amp;utm_campaign=Badge_Grade) [![Total alerts](https://img.shields.io/lgtm/alerts/g/Project-Lavinia/Lavinia-client.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-Lavinia/Lavinia-client/alerts/)
+
```

Comment:
Is this space added by you or by our formatting rules? Just curious.

Reply:
Yes, I did not manually change this, but Codacy seems to agree with the style.